From e61d5c472a5d0ad1f1a7ee7abf023e91ae43daae Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Thu, 27 Dec 2018 00:12:39 +0100
Subject: [PATCH] Appdata: remove old property

Use the non-legacy metainfo directory for the appdata file.
---
 gnucash/gnome/CMakeLists.txt         | 2 +-
 gnucash/gnome/gnucash.appdata.xml.in | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/gnucash/gnome/CMakeLists.txt b/gnucash/gnome/CMakeLists.txt
index 3e99c19bc..f47c9779d 100644
--- a/gnucash/gnome/CMakeLists.txt
+++ b/gnucash/gnome/CMakeLists.txt
@@ -180,7 +180,7 @@ endif()
 
 add_custom_target(gnucash-appdata ALL DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/gnucash.appdata.xml)
 
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/gnucash.appdata.xml DESTINATION  ${CMAKE_INSTALL_DATADIR}/appdata)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/gnucash.appdata.xml DESTINATION  ${CMAKE_INSTALL_DATADIR}/metainfo)
 
 #=======
 
diff --git a/gnucash/gnome/gnucash.appdata.xml.in b/gnucash/gnome/gnucash.appdata.xml.in
index 5318b7774..d82c786e2 100644
--- a/gnucash/gnome/gnucash.appdata.xml.in
+++ b/gnucash/gnome/gnucash.appdata.xml.in
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component>
-  <id type="desktop">org.gnucash.GnuCash</id>
+  <id>org.gnucash.GnuCash</id>
   <launchable type="desktop-id">gnucash.desktop</launchable>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
-- 
2.20.1

